app/js/lib/push_worker.js

```diff
@@ -28,7 +34,11 @@ self.addEventListener('push', function(event) {
     Promise.all([getMuteUntil(), getLastAliveTime()]).then(function (result) {
       var muteUntil = result[0]
       var lastAliveTime = result[1]
+<<<<<<< HEAD
```

Comment:
I didn't get how this happen. The PR seems to not touch any push-related activity.